model/ftn/ww3_shel.ftn

```diff
@@ -2206,7 +2226,7 @@
               IF ( FLH(J) ) THEN
                 CALL W3FLDH (J, NDST, NDSEN, NX, NY, NX, NY,    &
                              TIME0, TIMEN, NH(J), NHMAX, THO, HA, HD, HS,&
-                             TW0, WX0, WY0, DT0, TWN, WXN, WYN, DTN, IERR)
```

Comment:
Was this a bug? It looks like the winds were being read rather than ice concentration?

Reply:
Actually - I see you mentioned a bug in the homogenous ice generation in your currents, I assume it i that.

Reply:
Yes, that is the bug I mentioned.